mesonbuild/mintro.py

```diff
@@ -464,6 +472,7 @@ def run(options: argparse.Namespace) -> int:
     if 'meson.build' in [os.path.basename(options.builddir), options.builddir]:
         # Make sure that log entries in other parts of meson don't interfere with the JSON output
         with redirect_stdout(sys.stderr):
+            mesonlib.setup_vsenv()
```

Comment:
That's unrelated change, should be at least its own commit.

Reply:
Done. But it was needed if I wanted to be able to test it.